### DISCUSSION TITLE
关于动漫之家疑似停止更新并迁移至再漫画

### DISCUSSION THREAD

**Discussion by AiCrisia** [xiaoyaocz/flutter_dmzj#226] [General]
作者您好：
动漫之家似乎已经停止更新，9月10号更新了几部，9月11号只更新了两部，9月12号截止到晚上十点没有更新
有疑似原动漫之家的编辑以及一些汉化组的消息表明，动漫之家似乎已经失去维护，今后的上传会通过一个叫做“再漫画”的应用进行，动漫之家的数据库也会逐步迁移至“再漫画”
以下是我个人的需求
我已经使用您的客户端很久了，期间又产生大量的订阅和本地下载数据，如果一切尘埃落定，不知动漫之家X能否可以通过连接“再漫画”接口的方式达到无缝对接、而不必开发新的APP？

**Comment by xiaoyaocz** [comment]
懒得在一个APP里做兼容了，直接开了新分支，用再漫画的数据源搞了个新的APP
https://github.com/xiaoyaocz/flutter_dmzj/releases

**Comment by AiCrisia** [comment]
收到

**Comment by AiCrisia** [comment]
开发辛苦了！进去之后还是熟悉的界面，不过分类中少了一些tag，比如ゆり，也不能按照读者群和地域进行筛选，不太清楚是不是再漫画的的问题

**Comment by zsygood** [comment]
实际控制人圈钱，名义法定代表人也没钱，员工发不了工资，编辑走了。新网站目前没有广告。漫画更新有限
